File: NetFramework/Gauge/VB/GaugeDemo/PageKnob.vb
Imports C1.Win.C1Gauge

Public Class PageKnob

    Private Sub c1RadialGauge1_PointerDragMove(ByVal sender As System.Object, ByVal e As PointerDragEventArgs) Handles c1RadialGauge1.PointerDragMove
        e.Pointer.Value = e.NewValue
    End Sub

End Class

File: Core/Gauge/CS/GaugeDemo/PageKnob.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using C1.Win.Gauge;

namespace GaugeDemo
{
    public partial class PageKnob : BaseGaugePage
    {
        public PageKnob()
        {
            InitializeComponent();
        }

        private void c1RadialGauge1_PointerDragMove(object sender, PointerDragEventArgs e)
        {
            e.Pointer.Value = e.NewValue;
        }
    }
}
